diff --git a/node_modules/react-pageflip/build/html-flip-book/index.d.ts b/node_modules/react-pageflip/build/html-flip-book/index.d.ts
index a7da1b1..3ac2b3c 100644
--- a/node_modules/react-pageflip/build/html-flip-book/index.d.ts
+++ b/node_modules/react-pageflip/build/html-flip-book/index.d.ts
@@ -1,8 +1,8 @@
 import React from 'react';
 import { IFlipSetting, IEventProps } from './settings';
 interface IProps extends IFlipSetting, IEventProps {
-    className: string;
-    style: React.CSSProperties;
+    className?: string;
+    style?: React.CSSProperties;
     children: React.ReactNode;
     renderOnlyPageLengthChange?: boolean;
 }
diff --git a/node_modules/react-pageflip/build/html-flip-book/settings.d.ts b/node_modules/react-pageflip/build/html-flip-book/settings.d.ts
index 7246c82..5f87f3b 100644
--- a/node_modules/react-pageflip/build/html-flip-book/settings.d.ts
+++ b/node_modules/react-pageflip/build/html-flip-book/settings.d.ts
@@ -2,40 +2,40 @@ export declare type PageState = 'user_fold' | 'fold_corner' | 'flipping' | 'read
 export declare type PageOrientation = 'portrait' | 'landscape';
 export interface IFlipSetting {
     /** Page number from which to start viewing */
-    startPage: number;
+    startPage?: number;
     /** Whether the book will be stretched under the parent element or not */
-    size: 'fixed' | 'stretch';
-    width: number;
-    height: number;
-    minWidth: number;
-    maxWidth: number;
-    minHeight: number;
-    maxHeight: number;
+    size?: 'fixed' | 'stretch';
+    width?: number;
+    height?: number;
+    minWidth?: number;
+    maxWidth?: number;
+    minHeight?: number;
+    maxHeight?: number;
     /** Draw shadows or not when page flipping */
-    drawShadow: boolean;
+    drawShadow?: boolean;
     /** Flipping animation time */
-    flippingTime: number;
+    flippingTime?: number;
     /** Enable switching to portrait mode */
-    usePortrait: boolean;
+    usePortrait?: boolean;
     /** Initial value to z-index */
-    startZIndex: number;
+    startZIndex?: number;
     /** If this value is true, the parent element will be equal to the size of the book */
-    autoSize: boolean;
+    autoSize?: boolean;
     /** Shadow intensity (1: max intensity, 0: hidden shadows) */
-    maxShadowOpacity: number;
+    maxShadowOpacity?: number;
     /** If this value is true, the first and the last pages will be marked as hard and will be shown in single page mode */
-    showCover: boolean;
+    showCover?: boolean;
     /** Disable content scrolling when touching a book on mobile devices */
-    mobileScrollSupport: boolean;
+    mobileScrollSupport?: boolean;
     /** Set the forward event of clicking on child elements (buttons, links) */
-    clickEventForward: boolean;
+    clickEventForward?: boolean;
     /** Using mouse and touch events to page flipping */
-    useMouseEvents: boolean;
-    swipeDistance: number;
+    useMouseEvents?: boolean;
+    swipeDistance?: number;
     /** if this value is true, fold the corners of the book when the mouse pointer is over them. */
-    showPageCorners: boolean;
+    showPageCorners?: boolean;
     /** if this value is true, flipping by clicking on the whole book will be locked. Only on corners */
-    disableFlipByClick: boolean;
+    disableFlipByClick?: boolean;
 }
 export interface IBookState {
     page: number;
